### DISCUSSION TITLE
有没有大佬清楚，S912成功安装armbin系统最新版24 8.0版本后，插上网线无法获取到ip是什么原因，应该如何解决此问题。

### DISCUSSION THREAD

**Discussion by fxsion0621** [ophub/amlogic-s9xxx-armbian#2443] [Amlogic]
在Amlogic s912芯片的设备上成功安装了arebin系统后，插上网线无法获取到ip地址，使得无法获取到网络及其他的操作。还望有经验的大佬可以不吝赐教。

**Comment by ophub** [comment]
全部的s912设备都这样？还是哪个设备？你有没有根据首页的设备列表进去，看看之前分享者使用的安卓系统，先刷他分享的安卓，再安装armbian系统。

**Reply by fxsion0621** [reply]
好的，我再次尝试看看，感谢大佬答疑。

**Reply by fxsion0621** [reply]
已经修复了这个问题，是我路由器原因导致的，换了一个路由器就好了。不过最近有碰到一些问题，安装armbian系统内置的openwrt时总是报错，就算给网络加了魔法还是无法安装，还望大佬可以帮忙解决，更新docker时也会出现有几个连接无法连接。![mmexport1722179245497.png](https://github.com/user-attachments/assets/741d1146-6904-4fd4-95fc-08fa644c144b)
